hbase-balancer/src/main/java/org/apache/hadoop/hbase/master/balancer/BalanceAction.java

```diff
@@ -28,6 +28,8 @@ enum Type {
     ASSIGN_REGION,
     MOVE_REGION,
     SWAP_REGIONS,
+    ISOLATE_TABLE,
+    MOVE_BATCH,
```

Comment:
Our conditional candidate generators can frequently see pretty far into the future if they've gone to the trouble of deriving one very opinionated move anyway. So this is a nice way to represent multiple regions moves triggered by one candidate generation iteration